packages/web/src/workflow-trace-viewer/workflow-traces/trace-span-construction.ts

```diff
@@ -6,14 +6,13 @@ import { parseStepName, parseWorkflowName } from '@workflow/core/parse-name';
 import type { Event, Hook, Step, WorkflowRun } from '@workflow/world';
 import type { Span, SpanEvent } from '../trace-viewer/types';
 import { shouldShowVerticalLine } from './event-colors';
-import {
-  getResourceCode,
-  getResourceStatus,
-  getStatusResource,
-  WORKFLOW_LIBRARY,
-} from './trace-construction-types';
 import { calculateDuration, dateToOtelTime } from './trace-time-utils';
 
+const WORKFLOW_LIBRARY = {
+  name: 'vercel-workflow',
```

Comment:
what's this and can we get rid of it? preferably no references to "vercel" in the non-vercel world